pkg/networkserver/errors.go

```diff
@@ -49,6 +49,7 @@ var (
 	errRawPayloadTooShort                 = errors.Define("raw_payload_too_short", "length of RawPayload must not be less than 4")
 	errSchedule                           = errors.Define("schedule", "all downlink scheduling attempts failed")
 	errUnknownMACState                    = errors.DefineFailedPrecondition("unknown_mac_state", "MAC state is unknown")
+	errUnknownMACVersion                  = errors.DefineFailedPrecondition("unknown_mac_version", "MAC version is unknown")
```

Comment:
The lint action fails since the line is too long. We could probably add `//nolint:lll` before `var (` to ignore this warning for the whole code block. https://golangci-lint.run/usage/false-positives/#nolint-directive